includes/handler/class-quote-request.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * Handler for QuoteRequest activities.
+ *
+ * @package Activitypub
+ */
+
+namespace Activitypub\Handler;
+
+use Activitypub\Activity\Activity;
+use Activitypub\Collection\Actors;
+use Activitypub\Collection\Followers;
+use Activitypub\Collection\Remote_Actors;
+
+use function Activitypub\add_to_outbox;
+use function Activitypub\object_to_uri;
+
+/**
+ * Handler for QuoteRequest activities.
+ *
+ * @see https://codeberg.org/fediverse/fep/src/branch/main/fep/044f/fep-044f.md
+ */
+class Quote_Request {
+	/**
+	 * Initialize the class, registering WordPress hooks.
+	 */
+	public static function init() {
+		\add_action( 'activitypub_inbox_quoterequest', array( self::class, 'handle_quote_request' ), 10, 2 );
```

Comment:
I just realized we're running into the same hook name issue in the inbox. I think this is the first time we're handling a camelcase type? Maybe we could update the inbox to run `$type` through a snake_case function to create the hook name?